ToDoItem.py

```diff
@@ -0,0 +1,5 @@
+class ToDoItem:
```

Comment:
File name should be snake case: `to_do_item.py` to avoid confusion with classes.